libs/ledgerjs/packages/hw-app-kaspa/src/kaspaHwTransaction.ts

```diff
@@ -0,0 +1,244 @@
+type TransactionApiJSON = {
+  transaction: {
+    version: number;
+    inputs: TransactionInputApiJSON[];
+    outputs: TransactionOutputApiJSON[];
+    lockTime: number;
+    subnetworkId: string;
+  };
+};
+
+export class KaspaHwTransaction {
+  inputs: TransactionInput[];
+  outputs: TransactionOutput[];
+  version: number;
+  changeAddressType: number;
+  changeAddressIndex: number;
+  account: number;
+
+  constructor(txData: {
+    inputs: TransactionInput[];
+    outputs: TransactionOutput[];
+    version: number;
+    changeAddressType?: number;
+    changeAddressIndex?: number;
+    account?: number;
+  }) {
+    /**
+     * @type {TransactionInput[]}
+     */
+    this.inputs = txData.inputs;
+    /**
+     * @type {TransactionOutput[]}
+     */
+    this.outputs = txData.outputs;
+    /**
+     * @type {int}
+     */
+    this.version = txData.version;
+
+    this.changeAddressType = txData.changeAddressType ?? 0;
+    this.changeAddressIndex = txData.changeAddressIndex ?? 0;
+    this.account = txData.account ?? 0x80000000;
+
+    if (!(this.changeAddressType === 0 || this.changeAddressType === 1)) {
+      throw new Error(`changeAddressType must be 0 or 1 if set`);
```

Comment:
Backticks are not necessary with a string literal
```suggestion
      throw new Error("changeAddressType must be 0 or 1 if set");
```